pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>jv-join</artifactId>
+    <artifactId>jv-jdbc</artifactId>
```

Comment:
revert this change

Reply:
not fixed

Reply:
look into raw pom
<artifactId>jv-jdbc</artifactId> is present